content/administration/install/cdn.rst

```diff
@@ -4,77 +4,98 @@ Setting up a Content Delivery Network (CDN)
 
 .. _reference/cdn/keycdn:
 
-Deploying with KeyCDN_
-======================
+Deploying with KeyCDN
+=====================
 
-.. sectionauthor:: Fabien Meghazi
+.. sectionauthor:: Fabien Meghazi / Timothy Kukulka
 
-This document will guide you through the setup of a KeyCDN_ account with your
-Odoo powered website.
+A :abbr:`CDN (Content Delivery Network)` or Content Distribution Network, is a geographically
+distributed network of servers that provides high speed internet content. The
+:abbr:`CDN (Content Delivery Network)` provides quick, quality content for content heavy websites.
+
+This document will guide you through the setup of a KeyCDN_ account with an Odoo powered website.
 
 Step 1: Create a pull zone in the KeyCDN dashboard
 --------------------------------------------------
 
-.. image:: cdn/keycdn_create_a_pull_zone.png
-   :class: img-fluid
+On the KeyCDN dashboard, navigate to :guilabel:`Zones` menu item on the left. Name the
+:guilabel:`Zone Name`. This name will appear in the :abbr:`URL (Uniform Resource Locator)`. The
+:guilabel:`Zone Status` can be toggled back and forth from :guilabel:`Active` to
+:guilabel:`Inactive`. For this process the :guilabel:`Zone Type` should be :guilabel:`Pull`. Under
+:guilabel:`Pull Settings` enter the :guilabel:`Origin URL`. This should be the full database
+:abbr:`URL (Uniform Resource Locator)`.
 
-When creating the zone, enable the CORS option in the
-:guilabel:`advanced features` submenu. (more on that later)
+.. example::
+   Use ``https://yourdatabase.odoo.com``. Replace *yourdatabase* with the name of the database or the
+   custom URL.
 
-.. image:: cdn/keycdn_enable_CORS.png
-   :class: img-fluid
+.. image:: cdn/keycdn-zone.png
+   :align: center
+   :alt: KeyCDN's Zone configuration page.
 
-Once done, you'll have to wait a bit while KeyCDN_ is crawling your website.
+Under :guilabel:`General Settings` click the :guilabel:`Show all settings` button to expand the
+Zone options. This should be the last option on the page. After expanding the
+:guilabel:`General Settings` ensure that the :guilabel:`CORS` option is enabled. By default this
+option should be *enabled*.
 
-.. image:: cdn/keycdn_progressbar.png
-   :class: img-fluid
+Next, scroll to the bottom of the Zone configuration page and :guilabel:`Save` the changes. KeyCDN
+will indicate that the new Zone will be deployed. This can take about 10 minutes.
 
-.. note:: a new URL has been generated for your Zone, in this case it is
-          ``http://pulltest-b49.kxcdn.com``
+.. image:: cdn/zone-url.png
+   :align: center
+   :alt: KeyCDN deploying the new Zone.
 
-Step 2: Configure the odoo instance with your zone
---------------------------------------------------
+.. note::
+   A new :guilabel:`Zone URL` has been generated for your Zone, in this example it is
+   ``pulltest-xxxxx.kxcdn.com``. This will differ for each database.
+
+Copy this :guilabel:`Zone URL` to a Notepad as it will be used in the next steps.
```

Comment:
Notepad is a WIndows-specific program.
```suggestion
Copy this :guilabel:`Zone URL` to a text editor for later, as it will be used in the next steps.
```

Reply:
Good catch Zac!